### DISCUSSION TITLE
What's the right way to exit text editing ?

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5987] [Q&A] [answered]
A method like `component.setEditing(false)` would be ideal, but there is no such method.
Any advice?

**Comment by glaprida** [comment]
I tried doing:
`editor.getEditing().getEl().blur()`

and also
`editor.getEditing().getEl().setAttribute('contentEditable','false')`

but if I do `editor.getEditing()` after any of those (even if inside a timeout) it still returns the component.

**Reply by artf** [reply, marked as answer]
`editor.getEditing().trigger('disable')`
